### PR TITLE
build and install with miniconda

### PR DESCRIPTION
**Pull request by ErbB4**
from source

### CODE REVIEW

File: doc/install/build_install.rst [hunk 1]
```diff
@@ -960,3 +960,73 @@ need to be `updated <install-downloading_>`_.
...
+#redirect the libc files such that the miniconda environment can access it
+
+ln -sf /lib/x86_64-linux-gnu/libstdc++.so.6 /home/han/miniconda3/envs/arbor_test/bin/../lib/libstdc++.so.6
+
+
+#go to any working directory to try if you successfully installed arbor, by starting python and importing arbor
+
+python
+import arbor
+
```
Comment (thorstenhater) [inline doc/install/build_install.rst:1026]: One thing to add here could be testing for the version, i.e.
```
python -c 'import arbor; print(arbor.__version__)'
```
should work without errors and print something like `0.91-dev`.

File: doc/install/build_install.rst [hunk 2]
```diff
@@ -960,3 +960,73 @@ need to be `updated <install-downloading_>`_.
...
+python -c 'import numpy; print(numpy.__path__)'
+
+#load the right path to the one used for installing
+
+cp -r ~/miniconda3/envs/arbor_test/src/arbor/build/python/arbor <site-packages>
+Replace <site-packages> with the path you get in the previous operation before ‘/numpy’
+
+#redirect the libc files such that the miniconda environment can access it
+
+ln -sf /lib/x86_64-linux-gnu/libstdc++.so.6 /home/han/miniconda3/envs/arbor_test/bin/../lib/libstdc++.so.6
```
Comment (thorstenhater) [inline doc/install/build_install.rst:1019]:
```suggestion
ln -sf /lib/x86_64-linux-gnu/libstdc++.so.6 ~/miniconda3/envs/arbor_test/bin/../lib/libstdc++.so.6
```

File: doc/install/build_install.rst [hunk 3]
```diff
@@ -960,3 +960,73 @@ need to be `updated <install-downloading_>`_.
         git submodule update
     Or download submodules recursively when checking out:
         git clone --recurse-submodules https://github.com/arbor-sim/arbor.git
+
+
+Build in conda virtual environment using miniconda
+==================================================
+
+If you hope to install Arbor from source in a virtual environment in order not to interfere with other Arbor versions or software you have installed, or if you want to use features that are not included in the official release, you can use miniconda for this purpose. Otherwise, type pip install arbor in your terminal to install arbor. But you will have to fix two issues, one is feeding the correct Python path, and the other is 'GLIBCXX 3.4.30 not found in conda environment'.
```
Comment (thorstenhater) [inline doc/install/build_install.rst:968]: Maybe note that this is assuming Linux and MacOS is not yet covered.

File: doc/install/build_install.rst [hunk 4]
```diff
@@ -960,3 +960,73 @@ need to be `updated <install-downloading_>`_.
...
+mkdir build
+cd build
+
+cmake .. -GNinja -DCMAKE_CXX_COMPILER=$(which g++) -DCMAKE_C_COMPILER=$(which gcc) -DARB_WITH_PYTHON=ON -DARB_VECTORIZE=ON -DPython3_EXECUTABLE=$(which python3) -DARB_USE_BUNDLED_LIBS=ON
+
+
+#activate ninja to install
+
+ninja
+sudo ninja install
```
Comment (thorstenhater) [inline doc/install/build_install.rst:1004]: You don't need this and it'll install to the full system, i.e. outside any conda env, so maybe not recommend it ;)

File: doc/install/build_install.rst [hunk 5]
```diff
@@ -960,3 +960,73 @@ need to be `updated <install-downloading_>`_.
...
+
+#create a virtual environment
+
+conda create --name arbor_test
+conda activate arbor_test
+
+
+#go to the folder and clone the Arbor source package from GitHub
+
+cd ~/miniconda3/envs/arbor_test/
```
Comment (thorstenhater) [inline doc/install/build_install.rst:980]: This particular path is not required, a note towards might be helpful for a beginner. Something
like 'you can put this anywhere you like, but we chose this path.'

File: doc/install/build_install.rst [hunk 6]
```diff
@@ -960,3 +960,73 @@ need to be `updated <install-downloading_>`_.
...
+#go to any working directory to try if you successfully installed arbor, by starting python and importing arbor
+
+python
+import arbor
+
+#then deactivate the environment if no more actions are planned. In the future, always first activate the virtual environment with and then use arbor in this environment with:
+
+conda activate arbor_test
+python
+import arbor
```
Comment (thorstenhater) [inline doc/install/build_install.rst:1031]: you can add `>>>` here, which is the usual prompt from python to show that this command goes to the Python interpreter.

**Review by thorstenhater — CHANGES_REQUESTED** [review]

File: doc/install/build_install.rst (hunk 3 above)
Comment (ErbB4) [reply doc/install/build_install.rst]: added to the main text

File: doc/install/build_install.rst (hunk 6 above)
Comment (ErbB4) [reply doc/install/build_install.rst]: added. thanks for the advice.

**Review by thorstenhater — APPROVED** [review]
Thanks, look great

**Pull request merged**